2022-06-29 11:07:53,218 INFO    MainThread:4660 [wandb_setup.py:_flush():76] Loading settings from C:\Users\He Tao\.config\wandb\settings
2022-06-29 11:07:53,218 INFO    MainThread:4660 [wandb_setup.py:_flush():76] Loading settings from D:\Documents\codes\python\KnowledgeGraph\LinkPredict\SAC_KGR\src\wandb\settings
2022-06-29 11:07:53,218 INFO    MainThread:4660 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-29 11:07:53,218 INFO    MainThread:4660 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'experiments.py', 'program': 'D:/Documents/codes/python/KnowledgeGraph/LinkPredict/SAC_KGR/src/experiments.py'}
2022-06-29 11:07:53,218 INFO    MainThread:4660 [wandb_init.py:_log_setup():428] Logging user logs to D:\Documents\codes\python\KnowledgeGraph\LinkPredict\SAC_KGR\src\wandb\run-20220629_110753-34iottw3\logs\debug.log
2022-06-29 11:07:53,228 INFO    MainThread:4660 [wandb_init.py:_log_setup():429] Logging internal logs to D:\Documents\codes\python\KnowledgeGraph\LinkPredict\SAC_KGR\src\wandb\run-20220629_110753-34iottw3\logs\debug-internal.log
2022-06-29 11:07:53,229 INFO    MainThread:4660 [wandb_init.py:init():462] calling init triggers
2022-06-29 11:07:53,229 INFO    MainThread:4660 [wandb_init.py:init():466] wandb.init called with sweep_config: {}
config: {}
2022-06-29 11:07:53,229 INFO    MainThread:4660 [wandb_init.py:init():515] starting backend
2022-06-29 11:07:53,229 INFO    MainThread:4660 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-06-29 11:07:53,235 INFO    MainThread:4660 [backend.py:ensure_launched():217] starting backend process...
2022-06-29 11:07:53,398 INFO    MainThread:4660 [backend.py:ensure_launched():222] started backend process with pid: 17324
2022-06-29 11:07:53,399 INFO    MainThread:4660 [wandb_init.py:init():525] backend started and connected
2022-06-29 11:07:53,402 INFO    MainThread:4660 [wandb_init.py:init():596] updated telemetry
2022-06-29 11:07:53,404 INFO    MainThread:4660 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-06-29 11:08:23,409 INFO    MainThread:4660 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2022-06-29 11:08:23,409 ERROR   MainThread:4660 [wandb_init.py:init():636] backend process timed out
2022-06-29 11:08:23,409 ERROR   MainThread:4660 [wandb_init.py:init():647] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
